src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -186,24 +249,34 @@ const Comments = ({
                   </Formik>
                 ) : (
                   <>
-                    <p>{comment.content}</p>
-
-                    {canComment && user?.id === createdBy.id && (
+                    <p className="govuk-!-margin-top-3">{comment.content}</p>
+                    {canComment && (
                       <ButtonGroup>
-                        <ButtonText
-                          onClick={() => {
-                            setEditingComment(comment);
-                          }}
-                        >
-                          Edit
-                        </ButtonText>
+                        {user?.id === createdBy.id && (
+                          <>
+                            <ButtonText
+                              onClick={() => {
+                                setEditingComment(comment);
+                              }}
+                            >
+                              Edit
+                            </ButtonText>
+                            <ButtonText
+                              onClick={() => {
+                                removeComment(comment.id);
+                              }}
+                            >
+                              Delete
+                            </ButtonText>
+                          </>
+                        )}
 
                         <ButtonText
```

Comment:
Do we not need to show or hide this button based on permissions?

Reply:
Anyone who can edit the release should be able to resolve comments. So I believe this is correct as it is.